src/mmw/apps/user/views.py

```diff
@@ -108,12 +113,18 @@ def itsi_auth(request):
     user = authenticate(itsi_id=itsi_user['id'])
     if user is not None and user.is_active:
         auth_login(request, user)
-        return redirect('/')
+        redirect_uri = '{0}?{1}=true'.format(
```

Comment:
Is it okay to redirect from strings or should we use something more explicitly defined like `request.build_absolute_uri(reverse('itsi_login'))` or something?